templates/core/crowdfunding_donors.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'global/base.html' %}
+{% load staticfiles rev %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static_rev 'css/donors.css' %}">
+{% endblock %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-1"></div>
+    <div class="col-md-10">
+    <h2>Special thanks to all our amazing supporters!</h2>
+        <div class="row">
+        <div class="col-md-12">
+            <p>Thank you for supporting our crowdfunding campaign. Your support means the world to us.</p>
+        </div>
+        </div>
+
+        <div class="row">
+            <h2>They Support Us</h2>
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
+                <div class="col-md-4">
+                    <h4 class="donor-text">{{ donor.name }}</h4>
+                </div>
+                {% if forloop.counter|divisibleby:3 %}</div><div class="row">{% endif %}
+                {% if forloop.last %}</div>{% endif %}
```

Comment:
Ok, so what I propose for the quotes is the following:

1. Add file `quotes.py` under `core` directories and add hardcoded quotes like: 

```
DONORS_QUOTES = [
    {'text': '', 'author': '', 'url': ''},
    ...
]
```

and add a couple of quotes (with author and link if we could find a link to the tweet or something).

2. In `core/views.py` we should import this
```
from .quotes import DONOR_QUOTES
```
and later in the `crowdfunding_donors` function we could send it to the template:

```
return render(request, 'core/crowdfunding_donors.html', {        
    'donor_list': donor_list,
    'quotes': DONOR_QUOTES,
})
```

3. Create a `element_by_counter.py` file under `core/templatetags` with something like this:

```
from django import template
register = template.Library()

@register.simple_tag
def element_by_counter(data, counter, divider):
     index = int(counter / divider)
     if index < len(data):
          return data[index]
     return ''
```
4. In template add the file name to the `{% load %}` tag.

5. In template let's use it to display quote based on the counter we get from the outside loop.

```
{% if forloop.counter|divisibleby:XX}
    {% element_by_counter quotes forloop.counter XX as quote %}
    {{ quote.text }} <--- [let's use quote.text, quote.author and quote.url to style it here nicely]
{% endif %}
```

We should add tests of the new tag, too.